include/solvers/navier_stokes_scratch_data.h

```diff
@@ -108,6 +108,8 @@ class NavierStokesScratchData
     // By default, the assembly of variables belonging to auxiliary physics is
     // disabled.
     gather_VOF                   = false;
+    gather_pfg                   = false;
```

Comment:
write it verbosily : filtered_phase_gradient or phase_filtered_gradient
Something like that

Reply:
done.